backend/handler.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+const firebaseTokenVerifier = require('firebase-token-verifier')
```

Comment:
Once I got this working, it landed on L68 in this file and it threw an error that this package was missing. So I did `npm install --save firebase-token-verifier` in the backend folder

Reply:
I can't leave a comment on L68, so I'll leave it here. As is, the `/yourPins/categories` endpoint is still not working, and its throwing an error saying that `projectId is not defined`.

In this file, you'll need to define a variable `projectId` which corresponds to the ID of your Firebase project. You can also find this in `frontend/src/SignIn/signin.js` on L13. Your app should work after that change.

Reply:
Ohh I udnerstand now! THank you, that makes a lot of sense!!